pkg/prometheus/promcfg_test.go

```diff
@@ -825,11 +825,13 @@ func TestK8SSDConfigGeneration(t *testing.T) {
 	testcases := []struct {
 		apiServerConfig *monitoringv1.APIServerConfig
 		store           *assets.StoreBuilder
+		role            string
```

Comment:
```suggestion
		role            monitoringv1.ServiceDiscoveryRole
```

Reply:
Thank you for pointing this out. I have fixed this in [16dc1f3](https://github.com/prometheus-operator/prometheus-operator/pull/6672/commits/16dc1f3d8a62d9076512e7e47993d7153177c4a9)